components/admin_console/system_users/list/system_users_list.jsx

```diff
@@ -316,6 +318,8 @@ export default class SystemUsersList extends React.Component {
                     show={this.state.showManageTeamsModal}
                     onModalDismissed={this.doManageTeamsDismiss}
                     updateTeamMemberSchemeRoles={this.props.actions.updateTeamMemberSchemeRoles}
+                    getTeamsForUser={this.props.actions.getTeamsForUser}
```

Comment:
Can you make a `manage_teams_modal/index.jsx` and have that pass in the actions from there? The `SystemUserList` doesn't need to know about these actions

Reply:
Just for an action i don't think we should create a container. This does not cost us anything and feels like going against the general thumb rule of react components.

I.e create stateless first move to state if needed then to connected component if required.

Reply:
Makes sense. That does conflict with trying to reduce the coupling between those components, but I'm fine with leaving it as is for now.

Reply:
Now the modal has an index file so moved these functions into that